packages/administration/src/Component/Crud/Definition.php

```diff
@@ -24,15 +29,28 @@ public function __construct(
         public string $entityName,
         private CrudConfigData $config,
         private array $extensions,
+        private array $handlers,
     ) {
     }
 
     /**
-     * @return \Shopsys\AdministrationBundle\Controller\AbstractCrudControllerExtension[]
+     * @template T of object
+     * @param class-string<T>|null $interface
+     * @return ($interface is null
+     *     ? array<\Shopsys\AdministrationBundle\Controller\AbstractCrudControllerExtension>
+     *     : array<T&\Shopsys\AdministrationBundle\Controller\AbstractCrudControllerExtension&\Shopsys\AdministrationBundle\Component\Crud\Extension\CrudHookableExtensionInterface>
+     * )
      */
-    public function getExtensions(): array
+    public function getExtensions(?string $interface = null): array
```

Comment:
maybe this should be called `$hookableInterface` and annotated with `CrudHookableExtensionInterface` class-string?

Reply:
fixed